src/lib/pickles/requests.ml

```diff
@@ -114,6 +114,8 @@ module Step = struct
 
     type local_branches
 
+    type auxiliary_value
+
     type _ t +=
       | Compute_prev_proof_parts : prev_values H1.T(E01(Bool)).t -> unit t
```

Comment:
So this is arity zero to one.  I'm not sure, how but it could be clearer if it was just a little bit more self-explanatory, such as `E_arity01`.  Just trying to think of alternatives

Reply:
I like this a lot. Will try to include this in the other pickles renaming work!